### ISSUE TITLE
Add some help text to explain ORCID

### ISSUE THREAD

**Issue by arfon** [openjournals/theoj#28]
We're using ORCID but aren't saying what/why/how.

**Comment by arfon** [comment]
Depends upon https://github.com/openjournals/theoj/issues/29

**Comment by marcrohloff** [comment]
I'm probably not the best person to come up with the text :) If someone does I can drop it in quickly.

**Comment by freelanceastro** [comment]
Assuming this is a tooltip that shows up on hover over the "sign in" link. If so, here's some text:

"Sign in using your Open Researcher and Contributor ID (ORCID) credentials. If you don't have an ORCID, you'll be prompted to create one."

Sound good?

**Comment by arfon** [comment]
Sounds good. Minor quirk, it's actually an 'ORCID ID' (honestly) so I'd shorten this to:

> "Sign in using your ORCID ID. If you don't have an ORCID ID, you'll be prompted to create one."

**Comment by freelanceastro** [comment]
Oh jeez. Why did they do that? OK, yes, let's use that text.

**Comment by arfon** [comment]
Yeah, weird right :confused:
